cypress/integration/linkEmbedder.spec.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+describe('links to embeddable content', () => {
+  beforeEach(() => {
+    cy.visitTestEditor()
+  })
+
+  it('GitHub Gist', () => {
+    cy.codemirrorFill('https://gist.github.com/schacon/1')
+    cy.getMarkdownBody()
+      .find('span.one-click-embedding.gist-frame')
+      .click()
+    cy.getMarkdownBody()
+      .find('iframe')
+      .should('have.attr', 'title', 'gist schacon/1')
+  })
+
+  it('YouTube', () => {
+    cy.codemirrorFill('https://www.youtube.com/watch?v=YE7VzlLtp-4')
+    cy.getMarkdownBody()
+      .find('span.one-click-embedding.embed-responsive-item')
```

Comment:
Can you add a test for the one-click-embedding?

Reply:
What do you mean with this?

Reply:
That the one-click-embedding component reacts to clicks, renders a preview image, loads the content behind it, etc.

Reply:
If you look a few lines below your annotation, you'll find the click-reaction and content-loading. Just the preview image needs to be tested yet.

Reply:
Tests for preview images added